Lab-3.ipynb

```diff
@@ -0,0 +1,728 @@
+      "text/plain": [
+       "<Figure size 1080x504 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "x = np.linspace(10.1,13.8,100000)\n",
+    "g = stats.norm.pdf(x,loc=mu,scale=std)\n",
+    "\n",
+    "fig, (ax1,ax2) = plt.subplots(1, 2, figsize=(15,7),gridspec_kw={'wspace': 0.4})\n",
+    "ax1.set_title(\"Gaussian PDF Fit to Temperature Data\", fontsize=20)\n",
+    "ax2.set_title(\"Semilog Gaussian PDF Fit to Temperature Data\", fontsize=20)\n",
+    "ax1.set_xlabel(\"Average Temperature (K)\", fontsize=18), ax2.set_xlabel(\"Average Temperature (K)\", fontsize=18)\n",
+    "ax1.set_ylabel(\"Probability\", fontsize=18), ax2.set_ylabel(\"Log(Probability)\", fontsize=18)\n",
+    "ax1.set_xlim(8,15)\n",
+    "\n",
+    "ax1.plot(x,g, label=\"Gaussian PDF\", color='darkred', alpha=0.9, linewidth=5)\n",
+    "ax1.hist(d,100, density=True, label=\"Data\", color='cornflowerblue')\n",
+    "\n",
+    "ax2.plot(x,g, label=\"Gaussian PDF\", color='darkred', alpha=0.8, linewidth=5)\n",
+    "ax2.hist(d,100, density=True, label=\"Data\", color='cornflowerblue')\n",
+    "ax2.set_yscale('log')\n",
+    "\n",
+    "# guesses for the cutoff points\n",
+    "ax2.axvline(x=10.1, label='Cutoff', linestyle='--', linewidth=3, color='darkorange')\n",
+    "ax2.axvline(x=13.8, linestyle='--', linewidth=3, color='darkorange')\n",
+    "\n",
+    "ax1.tick_params(labelsize=16), ax2.tick_params(labelsize=16)\n",
+    "ax1.legend(loc='best'), ax2.legend(loc='best')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to answer the proposed statistical question, I began by plotting a Gaussian probability density function over the histogram of good and bad data, fit to the mean and standard deviations pulled from the histogram. In the left plot, it visually appears that an estimated Gaussian fits quite well with the boundaries I chose (x=10.1 to x=13.8). However, once overplotting on the semilog of the histogram data, we can see it is not as exact as it appeared. The left plot doesn't visually include the bad outliers, and the semilog plot shows that the PDF does not *exactly* fit.\n",
```

Comment:
Maybe I'm confused and cannot locate 1.a.3 in your lab, but don't you need to restate your question in math? Or is that you saying we should find the SD in relationship to the cutoff? Maybe make it clearer if so?